### PR TITLE
feat: `UseArrowFunctionsFixer` - add option `one_liners_only`

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: doc/rules/function_notation/use_arrow_functions.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Rule ``use_arrow_functions``
 ============================
 
-Anonymous functions with one-liner return statement must use arrow functions.
+Anonymous functions with return statement must use arrow functions.
```
Comment (keradus) [inline doc/rules/function_notation/use_arrow_functions.rst:5]:
```suggestion
Anonymous functions with return as the only statement must use arrow functions.
```

File: doc/rules/function_notation/use_arrow_functions.rst [hunk 2]
```diff
@@ -13,12 +13,26 @@ Using this rule is risky
 Risky when using ``isset()`` on outside variables that are not imported with
 ``use ()``.
 
+Configuration
+-------------
+
+``one_liners_only``
```
Comment (keradus) [inline doc/rules/function_notation/use_arrow_functions.rst:19]: what is the case we should not convert to arrow function if there is only one statement, the `return`, regardless how many lines it takes?

Comment (kubawerlos) [reply doc/rules/function_notation/use_arrow_functions.rst:19]: BC promise? Current description mentions "one-liner", are we ok to drop that limit without config option?

Comment (keradus) [reply doc/rules/function_notation/use_arrow_functions.rst:19]: I would love to better understand if there is a need for anyone to handle single-statement but multi-line differently.

if not, no need for option and we simply extended the fixer to touch case it was not touching yet - it's same style, applied for more valid elements.

if yes, ok to keep option.

I'm also OK to have it behind toggle and put v4 TODO to remove it

**Pull request closed without merging**